src/app/hearing-part/components/hearing-parts-preview/hearing-parts-preview.component.ts

```diff
@@ -64,9 +64,24 @@ export class HearingPartsPreviewComponent implements OnInit, OnChanges {
                 case 'reservedJudge':
                     return item[property] === undefined ? null : item[property].name;
 
+                case 'caseType':
+                    return item[property] === undefined ? null : item[property].description;
+
+                case 'hearingType':
+                    return item[property] === undefined ? null : item[property].description;
+
                 case 'priority':
                     return priorityValue(item[property]);
 
+                case 'scheduleStart':
+                    return item[property].toString();
+
+                case 'scheduleEnd':
+                    return item[property].toString();
```

Comment:
scheduleEnd is a date, when you do 'toString' on it, does sorting work as expected? I thought comparing dates will require some custom comparator.

Reply:
sorter, in the end, returns only string or a number, otherwise, it may work wrong. Having said that, you are right it shouldn't be simple .toString() here.